lib/components/resinio/sdk.js

```diff
@@ -134,9 +134,13 @@ exports.createSSHKey = async () => {
   }
 }
 
-exports.removeSSHKeys = () => {
-  return resin.models.key.getAll().each((key) => {
-    return resin.models.key.remove(key.id)
+exports.removeSSHKeyByLabel = (label) => {
```

Comment:
The create function is `.createSSHKey(label)` and takes a label. Can we make this `.removeSSHKey(label)` for symmetric purposes?